### ISSUE TITLE
[Suggestion] ‘Dustrial Decor

### ISSUE THREAD

**Issue by BiggieCheese026** [VazkiiMods/crucial2#218]
https://www.curseforge.com/minecraft/mc-mods/dustrial-decor
‘Dustrial Decor is a good vanilla-styled building mod and it’s really good for building cause it has a lot of cool blocks

**Comment by CheetahCyberBoi** [comment]
I agree. I feel like Quark's Iron Plates are just not enough for factory builds.
